docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,374 @@
+
+Below is a (uncomplete) set of services, that provide download or download and processing (marked with *) capabilities:
+
+See also a list of other places on [CSC research pages](https://research.csc.fi/open-gis-data#intdata3).
+
+=== "CSC * "
+
+    * Puhti
+        * [List of all available datasets in Puhti](../../../data/datasets/spatial-data-in-csc-computing-env.md)
+        * Sentinel and Landsat mosaics of Finland provided by FMI and SYKE: ```/appl/data/geo/sentinel/s2```
+        * Every CSC user has **read** access to data stored on Puhti, no need to move it, unless you need to modify it
+    * Allas
+        * [List of all available geospatial datasets in Allas](../../../data/datasets/spatial-data-in-csc-computing-env.md)
+        * Sentinel-2 L2A data of crop growing Finland, growing seasons 2016-present, [usage instructions](https://a3s.fi/sentinel-readme/README.txt)
+        * Data can be directly read from Allas without download for some cases, see eg [GDAL docs](../../../apps/gdal.md) and [Allas Python examples](https://github.com/csc-training/geocomputing/blob/master/python/allas/working_with_allas_from_Python_S3.py)
+        
+    
+=== "Open Access Hubs"
+
+    [SciHub](https://scihub.copernicus.eu/dhus/#/home)
+
+    * needs [registration](https://scihub.copernicus.eu/dhus/#/self-registration) 
+
+    * Sentinel 2 L1C and L2A products
+    * Sentinel 1 SLC, GRD , RAW and OCN products
+    * Worldwide
+    * Note: most of the data is in "Long term archive" and cannot be downloaded directly, but needs to be requested
+
+    [FinHub](https://finhub.nsdc.fmi.fi/#/home)
+
+    * Needs [registration](https://nsdc.fmi.fi/services/service_finhub_registration)
+    * Sentinel 2 L1C product
+    * Sentinel 1 SLC, GRD and OCN products
+    * Only Finland (and Baltics)
+
+    [ASF](https://search.asf.alaska.edu/#/)
+    
+    * Needs [registration](https://urs.earthdata.nasa.gov/users/new?)
+    * Sentinel 1 SLC, GRD , RAW and OCN products
+    * Many SAR and SAR derived datasets from other sensors
+    * Worldwide
+    * Sentinel 1 data available for immediate download
+    
+    **All of the above** provide a similar Graphical User Interface (GUI) and Application Programming Interface (API) to access the data.
+    Other tools for downloading the data from open access hubs: [sentinelsat](https://sentinelsat.readthedocs.io/en/stable/) with [examples for SciHub and FinHub](https://github.com/csc-training/geocomputing/blob/master/python/sentinel/sentinelsat_download_from_finhub_and_scihub.py), ...
+
+=== "USGS EarthExplorer"
+
+    [Earthexplorer](https://earthexplorer.usgs.gov/)
+
+    * Needs [registration](https://ers.cr.usgs.gov/register)
+    * Lots of different US related datasets 
+    * Main: Landsat worldwide
+    * GUI in web interface and bulk download
+    * Landsat download instructions: https://lta.cr.usgs.gov/sites/default/files/LS_C2_Help_122020.pdf
```

Comment:
Link doesn't work / redirects to https://www.usgs.gov/centers/eros/science/usgs-eros-archive-products-overview